assets/css/basicLayout.css

```diff
@@ -3,6 +3,7 @@
 */
 .header {
     background: #F8971D;
+    z-index: 999;
```

Comment:
Added this to ensure the headerAlt displays on top of all elements in it's view.